### PR TITLE
fix: add more zoom to map

### PR DESCRIPTION
**Pull request by meelrossi**
This pr does various fixes to map view in land section:
- Adds the possibility to zoom out more
- Fix click outside when closing filters popover in mobile
- Zoom to nearest land when filters are toggled

### CODE REVIEW

File: webapp/src/components/AssetBrowse/MapBrowse/utils.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export type Coord = {
+  x: number,
+  y: number
+}
+
+function getDistanceBetween(position1: Coord, position2: Coord) {
+  return Math.abs(position1.x - position2.x) + Math.abs(position1.y - position2.y)
+}
+
+export default function getNearestTile(currentPosition: Coord, coords: Array<Coord> = []) {
```
Comment (LautaroPetaccio) [inline webapp/src/components/AssetBrowse/MapBrowse/utils.ts:10]: Would you mind testing these functions?

File: webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.container.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  const wallet = getWallet(state)
+  const nftsOnRent = (wallet ? getOnRentNFTsByLessor(state, wallet?.address) : []).map(([nft]) => nft)
+
+  return {
+    tiles: getTiles(state),
+    ownedLands: [...nfts, ...nftsOnRent].filter(
+      nft =>
+        nft.category === NFTCategory.ESTATE ||
+        nft.category === NFTCategory.PARCEL
+    ),
```
Comment (LautaroPetaccio) [inline webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.container.ts:28]: The NFTs by lessors should always be either Estates or Parcels, is it needed to filter them?

File: webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useTabletAndBelowMediaQuery } from 'decentraland-ui'
+import { Atlas } from '../../Atlas'
+import { Props } from './MapBrowse.types'
+import { Coordinate } from 'recharts/types/util/types'
+import getNearestTile, { Coord } from './utils'
+import { NFTCategory } from '@dcl/schemas'
+
+export function MapBrowse({
```
Comment (LautaroPetaccio) [inline webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.tsx:9]: As this is a new component, is it worth it of being tested?

File: webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  useEffect(() => {
+    if (onlyOnRent && tilesForRent.length) {
+      const newCenter = getNearestTile({ x: x || 0, y: y || 0 }, tilesForRent)
+      if (newCenter) {
+        setX(newCenter.x)
+        setY(newCenter.y)
+      }
+    }
+    // this eslint rule is causing a warning due to x and y not being in the
+    // dependencies array. In this case, we don't won't this useEffect to be
```
Comment (LautaroPetaccio) [inline webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.tsx:82]:
```suggestion
    // dependencies array. In this case, we don't wan't this useEffect to be
```

**Review by LautaroPetaccio — COMMENTED** [review]
Great job! I've left some comments to discuss.

File: webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.tsx (hunk 3 above)
Comment (meelrossi) [reply webapp/src/components/AssetBrowse/MapBrowse/MapBrowse.tsx:9]: Created a task for this https://app.zenhub.com/workspaces/dapps-5ffc44ecec9f8500140e173c/issues/gh/decentraland/marketplace/1485 so we can add the tooling to test redux components

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**